components/List/index.jsx

```diff
@@ -0,0 +1,29 @@
+import WorkoutCard from "../Workouts/WorkoutCard";
+import useSWR from "swr";
+
+export default function List({ objectList }) {
```

Comment:
```suggestion
export default function List({ workouts }) {
```
`workouts` is clearer and semantically correct